### ISSUE TITLE
Отримання інформації про адресу земельної ділянки

### ISSUE THREAD

**Issue by Monstrofil** [ITDZK/api#1]
Доброго дня, чи можливо за допомогою API отримати інформацію про призначення та адресу земельної ділянки?

**Comment by newexception** [comment]
Нібито і відкрите api, але документація дуже слабка.

https://e.land.gov.ua/api/doc

**Comment by Monstrofil** [comment]
@newexception так, про це api я знаю, там є все що мені потрібно окрім адреси ділянки (накшталт "вулиця Шмідта 10"). Причому не видно великих проблем її туди додати, адже це наче не конфіденційна інформація.

@ITDZK чи дочекаємось ми тут коментаря (хоча б неофіційного), чи треба по-старому писати офіційні звернення поштою?

**Comment by newexception** [comment]
> @newexception так, про це api я знаю, там є все що мені потрібно окрім адреси ділянки (накшталт "вулиця Шмідта 10"). Причому не видно великих проблем її туди додати, адже це наче не конфіденційна інформація.
> 
> @ITDZK чи дочекаємось ми тут коментаря (хоча б неофіційного), чи треба по-старому писати офіційні звернення поштою?

Зрозуміло, якщо не вийде в api отримати, то маючи координати можна скористатися сторонніми сервісами реверсного геокодування

https://nominatim.org/release-docs/latest/api/Reverse/
https://developers.google.com/maps/documentation/geocoding/overview

**Comment by Monstrofil** [comment]
> Зрозуміло, якщо не вийде в api отримати, то маючи координати можна скористатися сторонніми сервісами реверсного геокодування

Це працює лише там, де є адресація. В моєму випадку я хочу адреси з кадастру навпаки, імпортувати в OSM (яким коритується і Nominatim).
